### ISSUE TITLE
Display issues on Firefox

### ISSUE THREAD

**Issue by crazoter** [getguesstimate/guesstimate-app#41]
tl;dr 
Unreadable description tag
No horizontal scrollbar
Display issues after scrolling horizontally with cursor keys

---

Using the public model [Drake Equation](http://getguesstimate.com/models/464) as an example, 
Upon entering the page:

![image](http://i.imgur.com/5mleS8C.png)

The description (or at least, I assumed it was) tag is scrunched up on the left. 
There was also no scrollbar to scroll left and right, so I had to select a cell and then press the left and right cursor keys to do so. 

When I tried to return (after having scrolled to the right) by pressing left however, this happened:

![image](http://i.imgur.com/7hFrj41.png)

I am unable to view the description and the header seems to have glitched. I was able to scroll all the way to the first cell though.

This was on Firefox 43.0.2.

P.S: Unrelated to the issue, I think it would help your app if you make it easier for users to provide feedback (e.g. "contact us", and/or "fork me on GitHub"). Good luck!

**Comment by OAGr** [comment]
Thanks for this.  I didn't do much testing in Firefox; most of my work was in Chrome.

I'll definitely work on this, but in the mean time, if you use Chrome things should work better.

**Comment by OAGr** [comment]
I got around to fixing the sidebar bug:
https://github.com/getguesstimate/guesstimate-app/commit/5b96085290905af2d32dd9cd2425f34413fd3feb

The scroll bar issue is annoying.  Will look into later on.
